src/test/java/nl/hsac/fitnesse/fixture/util/selenium/driverfactory/DriverManagerTest.java

```diff
@@ -0,0 +1,44 @@
+package nl.hsac.fitnesse.fixture.util.selenium.driverfactory;
+
+import nl.hsac.fitnesse.fixture.slim.StopTestException;
+import nl.hsac.fitnesse.fixture.util.selenium.SeleniumHelper;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.openqa.selenium.SessionNotCreatedException;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DriverManagerTest {
+
+    @Mock
+    private LocalDriverFactory driverFactory;
+
+    @Mock
+    private ChromeDriver webDriver;
+
+    @InjectMocks
+    private DriverManager driverManager;
+
+    @Test
+    public void getSeleniumHelperReturnsSeleniumHelper() {
+        when(driverFactory.createDriver()).thenReturn(webDriver);
+
+        SeleniumHelper result = driverManager.getSeleniumHelper();
+
+        Assert.assertNotNull(result);
+        Assert.assertEquals(result.driver(), webDriver);
+    }
+
+    @Test(expected = StopTestException.class)
```

Comment:
Given that you want a better message, shouldn't you test the message in the exception?

Reply:
looked up how to do that with junit, added. Have you ever considered using assertj? It's a much better library for assertions, and has a nice one for checking exceptions as well. I find the junit assertions a bit verbose.

Reply:
I've used assertj in the past and its assertions, especially for exceptions are much better that the standard jUnit stuff.  I tend not to use the jUnit `@Rule` stuff and just do a `try/catch` in the test method when just using jUnit.
Just like I didn't add mockito before: I never felt the complexity of the test warrant adding assertj, since I also did not feel like re-writing the existing tests. PRs to clean stuff up are always welcome.... ;-)